services/director-v2/src/simcore_service_director_v2/modules/dask_clients_pool.py

```diff
@@ -34,13 +33,9 @@ def __post_init__(self):
 
     @staticmethod
     def default_cluster(settings: ComputationalBackendSettings):
-        return Cluster(
-            id=settings.COMPUTATIONAL_BACKEND_DEFAULT_CLUSTER_ID,
-            name="Default cluster",
-            type=ClusterType.ON_PREMISE,
+        return DefaultCluster.construct(
```

Comment:
Aha! so there is a default cluster per app? is that the rationale?

Reply:
not sure I follow. I think you're idea of a constant is interesting, but since I still depend on the ENV variables, Maybe I should create the instance when starting the app. or maybe I could even cache it.

Reply:
As said above I think it is a bit misleading that a `DefaultObject` gets instantiated and receives some parameters which could change.

Reply:
I moved it into the app